values.yaml

```diff
@@ -224,6 +224,11 @@ persistence:
 
 imagePullPolicy: IfNotPresent
 
+# Use this set to assign a list of default pullSecrets
+imagePullSecrets: []
```

Comment:
I think we can remove the `[]` at the end of this line as users need to do that if they want to add a secret

Reply:
You're right. I'll remoe it.
Should I leave it as a comment?

Reply:
Ah, I should have read more thoroughly. I'll remove the `[]`